bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/observability/ObserveUtils.java

```diff
@@ -15,7 +15,6 @@
  *  specific language governing permissions and limitations
  *  under the License.
  */
-
```

Comment:
Missing a new line here

Reply:
Actually, most of the classes don't have a new line between header and package declaration. I removed some new lines to be consistent.